sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -1013,13 +1013,13 @@ class AnalysisSuite extends AnalysisTest with Matchers {
 
   test("SPARK-31975: Throw user facing error when use WindowFunction directly") {
     assertAnalysisError(testRelation2.select(RowNumber()),
-      Seq("Window function row_number() requires an OVER clause."))
+      Seq("Window function \"row_number()\" requires an OVER clause."))
```

Comment:
Please, migrate this onto `assertAnalysisErrorClass`

Reply:
Got it.